pages/glossary/large-scale-text.md

```diff
@@ -0,0 +1,9 @@
+---
+title: Large Scale Text
+key: large-scale-text
+---
+
+Text nodes with a [advance height](https://www.w3.org/TR/css-values/#length-advance-measure) of 18 [points](https://www.w3.org/TR/css-values/#pt), or 14 [points](https://www.w3.org/TR/css-values/#pt) and a [font weight](https://www.w3.org/TR/css-fonts-3/#font-weight-prop) is `bold`, `bolder` or a number of 700 or higher.
+
+**Note**: The advance height is the [computed](https://www.w3.org/TR/css-cascade-3/#computed) [font-size](https://www.w3.org/TR/css-fonts-3/#font-size-prop), with a [specified](https://www.w3.org/TR/css-cascade-3/#specified
```

Comment:
Why having this note here and not the same for advance height/width in the definition of background colors?